### PR TITLE
Allow local debugging with support repo tooling

### PR DESCRIPTION
**Pull request by jackwarner**
Look at me, @manwaring - getting some real-Odin under my belt. Now I can debug it locally using the tools at https://github.com/pariveda-serverless/support/tree/fix-local-docker-debug/nodejs/4.3.2/local

### CODE REVIEW

**Review by manwaring — COMMENTED** [review]
Is this local debugging tied to VSCode?  Thoughts on updating the README?

**Comment by jackwarner** [comment]
Good call - totally slipped my mind after I updated only the README in 'support'

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (manwaring) [inline .vscode/settings.json]: thoughts on gitignoring this file? seems like local dev preferences - is it needed for local debug?

Comment (jackwarner) [reply .vscode/settings.json]: Seems reasonable. As you can tell from my other gitignore commit, I'm an IntelliJ user so good to have some feedback from real VS Code users :)

**Review by manwaring — APPROVED** [review]

**Pull request merged**